chat/src/components/data-view/key-package.tsx

```
@@ -0,0 +1,12 @@
+import { type KeyPackage, type PrivateKeyPackage } from "ts-mls";
+import DataView from "./index";
```

Biome (2.1.2)

[error] Do not shadow the global "DataView" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)